packages/components/src/components/dropdown/dropdown.js

```diff
@@ -304,14 +310,14 @@ class Dropdown extends mixin(
         if (text) {
           text.innerHTML = itemToSelect.innerHTML;
         }
-        itemToSelect.classList.add(this.options.classSelected);
+        itemToSelect.parentElement.classList.add(this.options.classSelected);
```

Comment:
Do we need to choose `itemToSelect` vs. `itemToSelect.parentElement` conditionally, or does `classSelected` to `parentElement` work for older style, too?

Reply:
Because that class only does `display: none` it works with both HTML structures.